events.html.slim

```diff
@@ -19,8 +19,8 @@ description: Discover the events and webinars featuring talks about Red Hat prod
               img.desktop(src=cdn("#{site.base_url}/images/slider/RHDev_eventshero_desktop_1175x400_build2016.png") alt="Microsoft Build 2016")
           .slide
             a(href="http://www.devnation.org/")
-              img.mobile(src=cdn("#{site.base_url}/images/slider/RHDev_eventshero_devnation_mobile_01042016.png") alt="An Open Source Conference, By And For Developers Across The Globe")
-              img.desktop(src=cdn("#{site.base_url}/images/slider/RHDev_eventshero_devnation_desktop_01042016.png") alt="An Open Source Conference, By And For Developers Across The Globe")      
+              img.mobile(src=http://static.jboss.org/images/rhd/hero/RHDev_eventshero_mobile_devnation_03292016.png") alt="An Open Source Conference, By And For Developers Across The Globe")
+              img.desktop(src=http://static.jboss.org/images/rhd/hero/RHDev_eventshero_devnation_desktop_h250_03292015_.png") alt="An Open Source Conference, By And For Developers Across The Globe")      
```

Comment:
This should be 
img.desktop(src="http://static.jboss.org/images/rhd/hero/RHDev_eventshero_devnation_desktop_h250_03292015_.png" alt="An Open Source Conference, By And For Developers Across The Globe")